content/en/docs/reference/using-api/deprecation-policy.md

```diff
@@ -298,6 +298,18 @@ exist and function in releases up to and including X+8.  Only in release X+9,
 when API v1 has aged out, does the Widget resource cease to exist, and the
 behavior get removed.
 
+Starting in Kubernetes 1.19, making an API request to a deprecated REST API endpoint:
+
+1. returns a `Warning` header (as defined in [RFC7234, Section 5.5](https://tools.ietf.org/html/rfc7234#section-5.5)) in the API response.
+2. adds a `"k8s.io/deprecated":"true"` annotation to the [audit event](/docs/tasks/debug-application-cluster/audit/) recorded for the request.
+3. sets an `apiserver_requested_deprecated_apis` gauge metric to `1` in the `kube-apiserver` 
+   process. The metric has labels for `group`, `version`, `resource`, `subresource` that can be joined
+   to the `apiserver_request_total` metric, and a `removed_version` label that indicates the 
+   Kubernetes release in which the API will no longer be served. The following prometheus query
```

Comment:
nit (v1.19 vs. 1.19): Starting with Kubernetes v1.19 ...
nit: The following Prometheus query returns information ...

Reply:
> nit (v1.19 vs. 1.19): Starting with Kubernetes v1.19 ...

Happy to follow whatever convention the docs team wants, but there are 5 existing instances of "Starting in v1..." and 1 existing instance of "Starting in 1..." in the docs

> nit: The following Prometheus query returns information ...

done